charts/gardener/gardenlet/values.yaml

```
@@ -99,6 +99,7 @@ config:
       syncPeriod: 30s
       staleExtensionHealthChecks:
         enabled: true
+      managedResourceProgressingThreshold: 1h
       # threshold: 5m
```

The threshold field is under staleExtensionHealthChecks, hence:

```suggestion
      staleExtensionHealthChecks:
        enabled: true
      # threshold: 5m
      managedResourceProgressingThreshold: 1h
```